views/login.handlebars

```diff
@@ -1,7 +1,7 @@
 <nav class="navbar" role="navigation" aria-label="main navigation">
     <div class="navbar-brand">
         <a class="navbar-item" href="./index.html">
-            <img src="https://bulma.io/images/bulma-logo.png" width="112" height="28">
+            <img src="/assets/img/AskingForAFriend-Nav-Logo.png" width="25" height="25">
```

Comment:
Approving this, but it will be changed in my next pull request because I re-made the navbar logo